### PR TITLE
Task Keep browser open example added

### PR DESCRIPTION
**Pull request by ankitkumar734ac**
Example added.
Setting the detach parameter to true will keep the browser open after the driver process has been quit.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by harsha509 — APPROVED** [review]
Thank you @ankitkumar734ac !

**Comment by harsha509** [comment]
hi @ankitkumar734ac ,

Could you please sign CLA to get this merged ?

Thanks,
Sri

**Comment by ankitkumar734ac** [comment]
> hi @ankitkumar734ac ,
> 
> Could you please sign CLA to get this merged ?
> 
> Thanks,
> Sri

Already Done but it still show pending

**Comment by diemol** [comment]
Well, this user @kumarankit00792 still needs to sign it.

**Review by titusfortner — CHANGES_REQUESTED** [review]
Test method name is using Python convention not Java convention

**Comment by ankitkumar734ac** [comment]
> ###

Done

File: examples/java/src/test/java/dev/selenium/browsers/ChromeTest.java [hunk 1]
```diff
@@ -26,4 +26,11 @@ public void headlessOptions() {
         driver = new ChromeDriver(options);
     }
 
+    @Test
+    public void test_keep_browser_open() {
```
Comment (titusfortner) [inline examples/java/src/test/java/dev/selenium/browsers/ChromeTest.java]:
```suggestion
    public void keepBrowserOpen() {
```

**Comment by titusfortner** [comment]
Would also be helpful to add the example to the docs themselves in the 4 different translation files:

https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/website_and_docs/content/documentation/webdriver/browsers/chrome.en.md?plain=1#L121

(and thanks for helping us with this!)

**Review by ankitkumar734ac — COMMENTED** [review]
Method name changed as per suggestion

**Comment by ankitkumar734ac** [comment]
> Would also be helpful to add the example to the docs themselves in the 4 different translation files:
> 
> https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/website_and_docs/content/documentation/webdriver/browsers/chrome.en.md?plain=1#L121
> 
> (and thanks for helping us with this!)

Ok will try

**Comment by ankitkumar734ac** [comment]
Example added in the docs for 4 different translation files

**Comment by harsha509** [comment]
Looks good to me!

Thank you @titusfortner for the in-detail review.

**Review by harsha509 — APPROVED** [review]

**Pull request merged**